layer/filestore_unix.go

```diff
@@ -0,0 +1,13 @@
+// +build !windows
```

Comment:
I'm not sure I understand the reasoning behind making a unix only version that does nothing.